lua/lvim/core/theme.lua

```diff
@@ -85,8 +85,12 @@ M.setup = function()
   vim.g.colors_name = lvim.colorscheme
   vim.cmd("colorscheme " .. lvim.colorscheme)
 
-  require("lvim.core.lualine").setup()
-  require("lvim.core.lir").icon_setup()
+  if package.loaded.lualine then
```

Comment:
isn't it better to move this line into the plugin's `config` section instead?

https://github.com/LunarVim/LunarVim/blob/edc5e0e93f10cdd7ebd0de936b22ec4bfb2e852c/init.lua#L19

Reply:
I think it needs to be here so that it's run when the colorscheme is changed